### DISCUSSION TITLE
Any way to change date formatting in filenames?

### DISCUSSION THREAD

**Discussion by Pyccko** [DIGITALCRIMINALS/OnlyFans#396] [Q&A] [answered]
I'd like to be able to format as YYYY-MM-DD for easier sorting. Is this possible?

**Comment by P34k4k** [comment, marked as answer]
Yeah go to the config and change the following settings

"filename_format" "{date}_{filename}.{ext}",
"date_format": "%Y-%m-%d",

**Props to x34 for this info**

**Reply by Pyccko** [reply]
Thank you!
